static/js/resize.js

```diff
@@ -203,7 +203,10 @@ export function resize_bottom_whitespace(h) {
 export function resize_stream_filters_container(h) {
     h = narrow_window ? left_userlist_get_new_heights() : get_new_heights();
     resize_bottom_whitespace(h);
-    $("#stream-filters-container").css("max-height", h.stream_filters_max_height);
+    let height = h.stream_filters_max_height;
+    height = height - $("#private_messages").safeOuterHeight(true);
+    $("#stream-filters-container").css("max-height", height);
+    $("#private-container").css("max-height", h.stream_filters_max_height - 50);
```

Comment:
There's 1 thing in latest commit i would like to mention right now -> 

The issue reported for "Scrolling seems to be broken in `more conversations` view" was caused because i missed to allot a `max-height` CSS property to PM section, and hence the PMs went with `overflow:hidden`;

So now to fix it, we would need to allot a max-height here, for which i've used `50px` right now; which we might have to change/use a proper algorithm to calculate it, if we face any reported issues related to scrolling in more PMs section,

But for now it seemed to work properly in my local dev which is why i'm asking for a test-deploy of it; but yes, i won't be surprised with a reported bug on this.